chart/flux/templates/deployment.yaml

```diff
@@ -218,14 +218,17 @@ spec:
 {{ toYaml .Values.additionalArgs | indent 10 }}
           {{- end }}
           resources:
+{{ toYaml .Values.resources | indent 12 }}
+{{- if .Values.controller.extraContainers }}
```

Comment:
```suggestion
{{- if .Values.extraContainers }}
```

Reply:
done

Reply:
You fixed the line below but missed the one in the `if` statement :-)